tests/foreman/cli/test_docker.py

```diff
@@ -26,6 +30,27 @@
 
 EXTERNAL_DOCKER_URL = get_external_docker_url()
 INTERNAL_DOCKER_URL = get_internal_docker_url()
+STRING_TYPES = ['alpha', 'alphanumeric', 'cjk', 'utf8', 'latin1']
+
+
+def _make_docker_repo(product_id, name=None, upstream_name=None):
+    """Creates a Docker-based repository.
+
+    :param product_id: ID of the ``Product``.
+    :param str name: Name for the repository. If ``None`` then a random
+        value will be generated.
+    :param str upstream_name: A valid name for an existing Docker image.
+        If ``None`` then defaults to ``busybox``.
+    :return: A ``Repository`` object.
+
+    """
+    return make_repository({
+        'content-type': 'docker',
+        'docker-upstream-name': upstream_name or 'busybox',
```

Comment:
"busybox" and "docker" are magical values. They appear throughout this module, and if the value ever needs to change, we'll have to change every single appearance of the value. Can you define some module-wide constants and use them?

One nice approach could be to define relevant defaults like so:

```python
DOCKER_REPO = {
    'content-type': 'docker',
    'upstream-repository-name': 'busybox',
}
```

Or you could define several constants:

```python
REPO_CONTENT_TYPE = 'docker'
REPO_UPSTREAM_NAME = 'busybox'
```

Or something else. Go wild.